test/views/NinoExclusionSearchViewSpec.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+///*
+// * Copyright 2019 HM Revenue & Customs
+// *
+// * Licensed under the Apache License, Version 2.0 (the "License");
+// * you may not use this file except in compliance with the License.
+// * You may obtain a copy of the License at
+// *
+// *     http://www.apache.org/licenses/LICENSE-2.0
+// *
+// * Unless required by applicable law or agreed to in writing, software
+// * distributed under the License is distributed on an "AS IS" BASIS,
+// * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// * See the License for the specific language governing permissions and
+// * limitations under the License.
+// */
+
+package views
+
+import models._
+import org.jsoup.Jsoup
+import play.api.data.Form
+import play.twirl.api.Html
+import utils.FormMappings
+import views.helper.{PBIKViewBehaviours, PBIKViewSpec}
+
+
+class NinoExclusionSearchViewSpec extends PBIKViewSpec with FormMappings {
+
+  def taxYearRange = TaxYearRange(2018, 2019, 2020)
+  val messageKeyPrefix = "ninoExclusionSearchForm"
+
+  override def view: Html = viewWithForm(exclusionSearchFormWithoutNino)
+
+  def viewWithForm(form: Form[EiLPerson]): Html =
+    views.html.exclusion.ninoExclusionSearchForm(taxYearRange, "cyp1", "30", form, true, EmpRef("", ""))
+
+
+  "exclusionNinoOrNoNinoForm" must {
```

Comment:
Name of test is a little off. Maybe `ninoExclusionSearchPage` would be better

Reply:
Done